### ISSUE TITLE
Broken after update to Atom 0.205.0?

### ISSUE THREAD

**Issue by we3geeks** [atom/tree-view#462]
Treeview plugin no longer works after updating to Atom v0.205.0 this morning. I can't get it to open using either the assigned keybinding (cmd-\) or via the "Packages > Treeview > Toggle" menu entry.

**Comment by mnquintana** [comment]
Can you please follow the steps in the [debugging document](https://atom.io/docs/latest/hacking-atom-debugging)? Can you reproduce this in safe mode? (`atom --safe`)

**Comment by we3geeks** [comment]
`atom --version` shows "0.205.0"

`apm links` shows no links under either ~/.atom/dev/packages or ~/.atom/packages

"Key Binding Resolver" shows the following for cmd-\:

![atom-20150603](https://cloud.githubusercontent.com/assets/2390335/7965105/ab71eaac-09d8-11e5-9778-b5654fcd6c21.png)

No errors shown in dev tools, other than a warning from the Emmet plugin (which doesn't seem relevant?): 

![atom-20150603b](https://cloud.githubusercontent.com/assets/2390335/7965240/7b669d70-09d9-11e5-8800-a316d5f46e22.png)

Same behavior under `atom --safe`

I'm on Mac OS X 10.10.3, if that matters.

Let me know if there is anything more I can provide to help track this down.

**Comment by lee-dohm** [comment]
How did you open Atom? From the command-line? Or from the icon? If from the command-line, what command did you give?

**Comment by mnquintana** [comment]
Actually, what opened in Atom? Was it just a blank unsaved buffer? (e.g. `untitled`) If so, Atom probably opened not in any directory, which means you wouldn't be able to open the Tree View (since there's no directory to explore). If that's the case, to "fix" this, just open a directory in Atom. There's a related issue you might want to follow: https://github.com/atom/tree-view/issues/68

**Comment by we3geeks** [comment]
Ah, good catch @mnquintana 

This seems to have something to do with the restart after updating? Atom has recently remembered where (i.e.g, which directory) I last had open even if I didn't necessarily have any files from that folder open when I exited and restarted. I just opened a folder, and the tree view folder now works (I can toggle it open and closed as expected). And after exiting Atom, even with just the untitled buffer open, I can toggle tree view without any problem as expected.

I'm inclined to chalk this off as something related to the restart-after-updating for this version.
